gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/service/tasks/LaunchConfig.java

```diff
@@ -208,12 +214,25 @@ static void action(Params params) {
                         .classpath(relativizeToServiceLibDirectory(params.getClasspath()))
                         .addAllJvmOpts(javaAgentArgs(params))
                         .addAllJvmOpts(alwaysOnJvmOptions)
+                        .addAllJvmOpts(avxOptions)
                         .addAllJvmOpts(params.getDefaultJvmOpts().get())
                         .env(defaultEnvironment)
                         .build(),
                 params.getCheckLauncher().get().getAsFile());
     }
 
+    // When a specific jdk is provided, we can assume a modern versions including the
```

Comment:
I suppose it's possible for us to have a modern jdk bundled, but have an affected java version? If not, I think we can remove the assumption by having our condition below be: `!params.getBundledJdks().get || (11 <= javaVersion <= 19)`?

Reply:
we recall products which don't stay up to date, so they're guaranteed to have reasonable jdks bundled :-]